scripts/fix-install-names.py

```diff
@@ -13,6 +14,9 @@ def main(argv=tuple(sys.argv[1:])):
 
     bundle_path = Path(arguments.bundle[0])
     framework_path = bundle_path / 'Contents' / 'Frameworks'
+    rpath_str = '/usr/local/lib'
+    if platform.machine() == 'arm64':
+        rpath_str = '/opt/homebrew/lib'
```

Comment:
use `brew --prefix` to get homebrew install prefix ?

Reply:
This was initially written for use cases when homebrew is not required(as the old Intel install script does not use homebrew's qt). Probably we can do this for now as the Intel build has to use homebrew's qt as well for now.